### ISSUE TITLE
Time Picker - Provide a valid label for form fields - (2036570363)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5610]
### Module: 
 35 Time Picker 

### Violation: 
 Provide a valid label for form fields 

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036570363 

### Severity: 
 10 

### Description: 
 [Issue]
There are buttons with incomplete and identical accessible names. Examples include:

- Increase hour
- hour
- Decrease hour
- Increase minute
- Minute
- Decrease minute
- Increase AM/PM
- AM/PM
- Decrease AM/PM

[User Impact]
Screen reader users will be unable to determine the purpose of these buttons. Speech input users will have difficulty activating them.

[Code Reference] 
 ```
<span aria-label="Increase hour" class="button button--hour-up button--top-left" role="button" tabindex="-1">
  <calcite-icon aria-hidden="true" icon="chevron-up" scale="s" calcite-hydrated=""></calcite-icon>
</span>
 ``` 

### Note: 
 [Suggestion]
Provide a valid label for form fields. This includes providing accessible names for buttons. The accessible name of a button can be set with an aria-label attribute. Good accessible names are both concise and descriptive.

[Compliant Code Example] 
 ```
<span aria-label="Small scale Increase hour" class="button button--hour-up button--top-left" role="button" tabindex="-1">
  <calcite-icon aria-hidden="true" icon="chevron-up" scale="s" calcite-hydrated=""></calcite-icon>
</span>
 ```
 

### Media Type: 
 Forms 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This surfaced as a result of the testing page and is not a valid bug. Closing.
